packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesFileDownloader.m

```diff
@@ -205,11 +205,22 @@ - (void)_createUpdateWithManifest:(NSMutableDictionary *)mutableManifest
   }
 
   NSError *error;
-  EXUpdatesUpdate *update = [EXUpdatesUpdate updateWithManifest:mutableManifest.copy
-                                                       response:response
-                                                         config:_config
-                                                       database:database
-                                                          error:&error];
+  EXUpdatesUpdate *update;
+  @try {
```

Comment:
This boundary seems much more targeted in scope than the one above (which is good, just pointing it out to reference my above comment about narrowing scope)